package.json

```diff
@@ -50,7 +50,7 @@
     "eslint": "^3.19.0",
     "eslint-config-webpack": "^1.2.1",
     "eslint-plugin-import": "^2.2.0",
-    "jest": "^19.0.2",
+    "jest": "^20.0.3",
     "lint-staged": "^3.4.0",
     "nodemon": "^1.11.0",
```

Comment:
The other open pull request bumps `babel-preset-env` to `1.5.0` which is the only version discrepancy that I can see between #49 and the `master` this was based off of.